src/crypto/scalarmult/curve25519.rs

```diff
@@ -26,12 +26,13 @@ new_type! {
 /// `Ok(q)`.
 /// If the the `GroupElement` is all zero, `scalarmult()` returns `Err(())` since
 /// the resulting `GroupElement` would be all zero, no matter the `Scalar`.
-pub fn scalarmult(&Scalar(ref n): &Scalar,
-                  &GroupElement(ref p): &GroupElement)
-                  -> Result<GroupElement, ()> {
+pub fn scalarmult(
```

Comment:
There are a lot of whitespace changes here that complicate reviewing the diff, especially since we have a bunch of test vectors here, would be better if this file didn't have these whitespace changes.

Reply:
I agree with this... maybe introduce a separate PR before this PR that runs the `rustfmt`?